make updatedeps fails on master with go 1.5 beta 3

Hi,

I wanted to test the latest master with go 1.5 beta 3 (same happens with beta2) and I've run into a problem with fetching the dependencies.

```
florin@sniper [02:04:15] [~/golang/src/github.com/hashicorp/terraform] [master]
-> % make updatedeps
go get -u github.com/mitchellh/gox
go get -u golang.org/x/tools/cmd/stringer
go list ./... \
        | xargs go list -f '{{join .Deps "\n"}}' \
        | grep -v github.com/hashicorp/terraform \
        | sort -u \
        | xargs go get -f -u -v
code.google.com/p/go-uuid (download)
warning: code.google.com is shutting down; import path code.google.com/p/go-uuid/uuid will stop working
github.com/armon/circbuf (download)
github.com/aws/aws-sdk-go (download)
github.com/vaughan0/go-ini (download)
package github.com/aws/aws-sdk-go/internal/protocol/ec2query
        imports github.com/aws/aws-sdk-go/internal/protocol/ec2query: use of internal package not allowed
package github.com/aws/aws-sdk-go/internal/protocol/json/jsonutil
        imports github.com/aws/aws-sdk-go/internal/protocol/json/jsonutil: use of internal package not allowed
package github.com/aws/aws-sdk-go/internal/protocol/jsonrpc
        imports github.com/aws/aws-sdk-go/internal/protocol/jsonrpc: use of internal package not allowed
package github.com/aws/aws-sdk-go/internal/protocol/query
        imports github.com/aws/aws-sdk-go/internal/protocol/query: use of internal package not allowed
package github.com/aws/aws-sdk-go/internal/protocol/query/queryutil
        imports github.com/aws/aws-sdk-go/internal/protocol/query/queryutil: use of internal package not allowed
package github.com/aws/aws-sdk-go/internal/protocol/rest
        imports github.com/aws/aws-sdk-go/internal/protocol/rest: use of internal package not allowed
package github.com/aws/aws-sdk-go/internal/protocol/restjson
        imports github.com/aws/aws-sdk-go/internal/protocol/restjson: use of internal package not allowed
package github.com/aws/aws-sdk-go/internal/protocol/restxml
        imports github.com/aws/aws-sdk-go/internal/protocol/restxml: use of internal package not allowed
package github.com/aws/aws-sdk-go/internal/protocol/xml/xmlutil
        imports github.com/aws/aws-sdk-go/internal/protocol/xml/xmlutil: use of internal package not allowed
package github.com/aws/aws-sdk-go/internal/signer/v4
        imports github.com/aws/aws-sdk-go/internal/signer/v4: use of internal package not allowed
github.com/awslabs/aws-sdk-go (download)
github.com/Azure/azure-sdk-for-go (download)
github.com/Azure/go-pkcs12 (download)
github.com/cyberdelia/heroku-go (download)
github.com/dylanmei/iso8601 (download)
github.com/dylanmei/winrmtest (download)
github.com/masterzen/winrm (download)
github.com/masterzen/simplexml (download)
github.com/masterzen/xmlpath (download)
github.com/satori/go.uuid (download)
github.com/fsouza/go-dockerclient (download)
github.com/hashicorp/atlas-go (download)
github.com/hashicorp/go-version (download)
github.com/hashicorp/consul (download)
github.com/hashicorp/errwrap (download)
github.com/hashicorp/go-checkpoint (download)
github.com/hashicorp/go-multierror (download)
github.com/hashicorp/hcl (download)
github.com/hashicorp/yamux (download)
github.com/imdario/mergo (download)
github.com/nu7hatch/gouuid (download)
github.com/mitchellh/cli (download)
golang.org/x/crypto (download)
github.com/mitchellh/colorstring (download)
github.com/mitchellh/copystructure (download)
github.com/mitchellh/reflectwalk (download)
github.com/mitchellh/go-homedir (download)
github.com/mitchellh/go-linereader (download)
github.com/mitchellh/mapstructure (download)
github.com/mitchellh/osext (download)
github.com/mitchellh/packer (download)
github.com/mitchellh/panicwrap (download)
github.com/mitchellh/prefixedio (download)
github.com/packer-community/winrmcp (download)
github.com/pearkes/cloudflare (download)
github.com/pearkes/digitalocean (download)
github.com/pearkes/dnsimple (download)
github.com/pearkes/mailgun (download)
github.com/rackspace/gophercloud (download)
github.com/soniah/dnsmadeeasy (download)
github.com/xanzy/go-cloudstack (download)
golang.org/x/net (download)
golang.org/x/oauth2 (download)
google.golang.org/cloud (download)
google.golang.org/api (download)
Makefile:43: recipe for target 'updatedeps' failed
make: *** [updatedeps] Error 123

florin@sniper [02:05:06] [~/golang/src/github.com/hashicorp/terraform] [master]
-> % go env
GOARCH="amd64"
GOBIN=""
GOEXE=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOOS="linux"
GOPATH="/home/florin/golang"
GORACE=""
GOROOT="/home/florin/IdeaProjects/go"
GOTOOLDIR="/home/florin/IdeaProjects/go/pkg/tool/linux_amd64"
GO15VENDOREXPERIMENT=""
CC="gcc"
GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0"
CXX="g++"
CGO_ENABLED="1"

florin@sniper [02:05:19] [~/golang/src/github.com/hashicorp/terraform] [master]
-> % go version
go version devel +d3ffc97 Wed Jul 29 23:50:20 2015 +0000 linux/amd64
``` 

Kind regards,
Florin

--- a/Makefile
+++ b/Makefile
@@ -45,6 +45,7 @@ updatedeps:
 	go list ./... \
 		| xargs go list -f '{{join .Deps "\n"}}' \
 		| grep -v github.com/hashicorp/terraform \
+		| grep -v '/internal/' \
 		| sort -u \
 		| xargs go get -f -u -v
 

